repos/fdbt-netex-output/src/netex-convertor/point-to-point-tickets/pointToPointTicketNetexHelpers.ts

```diff
@@ -490,9 +491,57 @@ export const combineFareZones = (outbound: FareZone[], inbound: FareZone[]): Far
 export const getLinesElement = (
     ticket: PointToPointTicket | PointToPointPeriodTicket,
     lineName: string,
+    isReturnAndHasAdditionalService: boolean,
 ): NetexObject => {
     const typeOfPointToPoint = ticket.type;
 
+    if (!isReturnAndHasAdditionalService) {
+        return {
+            version: '1.0',
+            id: `Tariff@${typeOfPointToPoint}@lines`,
+            Name: { $t: `O/D pairs for ${lineName}` },
+            TypeOfFareStructureElementRef: {
+                versionRef: 'fxc:v1.0',
+                ref: 'fxc:access',
+            },
+            distanceMatrixElements: {
+                DistanceMatrixElement: isReturnTicket(ticket)
+                    ? getDistanceMatrixElements(combineFareZones(ticket.outboundFareZones, ticket.inboundFareZones))
+                    : getDistanceMatrixElements(ticket.fareZones),
+            },
+            GenericParameterAssignment: {
+                version: '1.0',
+                order: '01',
+                id: `Tariff@${typeOfPointToPoint}@lines`,
+                TypeOfAccessRightAssignmentRef: {
+                    version: 'fxc:v1.0',
+                    ref: 'fxc:can_access',
+                },
+                ValidityParameterAssignmentType: { $t: 'EQ' },
+                validityParameters: {
+                    LineRef: {
+                        version: '1.0',
+                        ref: lineName,
+                    },
+                },
+            },
+        };
+    }
+
+    const lineRefs = [
+        {
+            version: '1.0',
+            ref: lineName,
+        },
+        {
+            version: '1.0',
+            ref:
+                isReturnTicket(ticket) && ticket.additionalServices && ticket.additionalServices.length > 0
```

Comment:
If we're checking if its a return, and it has additional services here, I dont think it makes much sense using the isReturnAndHasAdditionalService which is passed into this function, as we're not really using it.

Reply:
I think this check can be pushed to the top of this function, i.e.

``

if ( isReturnTicket(ticket) && ticket.additionalServices && ticket.additionalServices.length > 0 )

and we can remove the need for isReturnAndHasAdditionalService

Reply:
@NitishAroraKPMG

Reply:
moved the check to head of file